_locales/en/messages.json

```diff
@@ -750,9 +750,6 @@
 	"panel_detail_performance_text": {
 		"message": "Feature coming soon."
 	},
-	"panel_detail_rewards_title": {
```

Comment:
Did you run the test script?  I'm still seeing this string in some snapshot files

Reply:
Hmm I'm seeing that the unit tests do pass but I see `Rewards.jsx.snap`.

Reply:
The test is skipped so the unit tests pass but the snapshot is still out of date.  Did you skip the test?

Reply:
git blame says I did, back in March. It must have been a quick workaround that we needed for the dynamic UI testing that I did not revert.

Reply:
Made changes necessary to reenable the Rewards component tests, reenabled them, and updated the snapshots.